flax/linen/recurrent.py

```diff
@@ -392,6 +392,113 @@ def num_feature_axes(self) -> int:
     return 1
 
 
+class SimpleCell(RNNCellBase):
+  r"""Simple cell.
+
+  The mathematical definition of the cell is as follows
+
+  .. math::
+
+      \begin{array}{ll}
+      h' = \tanh(W_i x + b + W_h h)
```

Comment:
Also consider adding a separate math block denoting the equation if `self.residual=True`; i.e. `h' = \tanh(W_i x + b_i + W_h h + h)`